### PR TITLE
Create a data model

### PR DESCRIPTION
**Pull request by utsargo**
### Description
- Create a new app named `rails-blog`
- Initialize the project with Git.
- Set up Postgres database.
- Build the project schema.
    - Create and run the necessary migration files.
    - Table and column names match the ERD diagram.
    - Foreign keys are included.
    - All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: db/migrate/20230405123925_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.string :Bio
+      t.datetime :UpdatedAt
+      t.datetime :CreatedAt
+      t.integer :PostsCounter
+      t.timestamps
```
Comment (emmanuellekamwa) [inline db/migrate/20230405123925_create_users.rb:10]: - [x] So as to respect the rails syntax, words like `posts_counter`, `updated_at`, and `created_at` are supposed to be written using Camel case syntax just as I did. Please kindly fix that.

File: db/migrate/20230405125230_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  def change
+    create_table :posts do |t|
+      t.integer :AuthorId
+      t.string :Title
+      t.string :Text
+      t.datetime :CreatedAt
+      t.datetime :UpdatedAt
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
+      t.timestamps
```
Comment (emmanuellekamwa) [inline db/migrate/20230405125230_create_posts.rb:11]: - [x] The same error is seen here. Please kindly fix that.

File: db/migrate/20230405124908_create_comments.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :AuthorId
+      t.integer :PostId
+      t.string :Text
+      t.datetime :UpdatedAt
+      t.datetime :CreatedAt
+      t.timestamps
```
Comment (emmanuellekamwa) [inline db/migrate/20230405124908_create_comments.rb:9]: - [x] Same thing here.

File: db/migrate/20230405125422_create_likes.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :AuthorId
+      t.integer :PostId
+      t.datetime :CreatedAt
+      t.datetime :UpdatedAt
+      t.timestamps
+    end
```
Comment (emmanuellekamwa) [inline db/migrate/20230405125422_create_likes.rb:9]: - [x] Same syntax error here.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ⛔🛑❌**
Hello @utsargo 👋
Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights 
- Descriptive PR 
- Linters are passing all the checks.

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @utsargo,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Detailed pull request message ✔️
- Correct work flow was used ✔
approval1
️

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**